jalib/jfilesystem.cpp

```diff
@@ -206,9 +208,11 @@ jalib::Filesystem::GetProgramName()
 jalib::string
 jalib::Filesystem::GetProgramPath()
 {
-  static jalib::string value = _GetProgramExe();
-
-  return value;
+  static jalib::string *value = NULL;
+  if (value == NULL) {
```

Comment:
Suggest adding comment:
// Technically, this is a memory leak, but `s` is static and so it happens only once.